ccp/lib/clients/openshift/client.py

```diff
@@ -0,0 +1,246 @@
+from ccp.lib.clients.base import CmdClient
+from ccp.lib.utils.retry import retry
+from os import path
+
+
+class OpenShiftCmdClient(CmdClient):
+    """
+    Client for interacting with Openshift.
+    """
+
+    def __init__(self):
+        super(OpenShiftCmdClient, self).__init__(
+            "/usr/bin/oc"
+        )
+
+    @retry(tries=10, delay=3, backoff=2)
+    def get_sa_token_from_openshift(self, namespace, sa="sa/jenkins"):
```

Comment:
There's a similar code at https://github.com/CentOS/container-pipeline-service/blob/ef004815ef119f8619ed91657de3dbf1aca15a49/ccp/lib/openshift.py#L14-L31

Can we reuse/modify that and get things done instead?

Reply:
I have kept the file so as to not break anything. Once index reader is refactored, this file can be dropped.

Reply:
agree with @dharmit, that makes code better maintainable

Reply:
I request to do the necessary refactoring in this iteration itself. If something's not looking good, let's talk and fix it. Because piling things up is going to lead us to a round of refactoring that will require all of us to work on it for a sprint or two.

Reply:
OK,  I will get that done then

Reply:
Done. For now i have updated it to use OpenshiftCmdClient to get the jenkins token.. 

Note the file will be removed after the Openshift Jenkins Processor is merged. That bit is outside the scope of this PR.